_posts/2017-07-25-oes-case-study.md

```diff
@@ -0,0 +1,41 @@
+
+**Standards team:** How did you hear about the U.S. Web Design Standards?
+
+**Maureen Earley:**  Members of our team  worked with the 18F Federal Front Door team that was involved in creating the Standards. The Office of Evaluation Sciences is a champion of the Standards because we’re committed to making government easier for people to interact with. We also wanted to make sure our own website was an ambassador of the Standards and all that they stand for.
+
+**Standards team:** Why did you decide to use the Standards?
+
+**Maureen Earley:** We understood why the Standards were created and why good design across government is important. We also didn’t want to recreate the wheel in designing our website. For example, we realized during the design process that our OES logo didn’t meet color contrast requirements for accessibility. But, because we were referencing the Standards we were able to avoid this issue. We wanted to defer to the Standards and the expertise that already exists.
+
+We also have been working with the team running [Federalist](https://federalist.fr.cloud.gov), the open source static website publishing service, building out templates that other teams across government can use. For example, our [Team page](https://oes.gsa.gov/team/) is now a Federalist template that anyone can use.
+
+When we talk with other offices within government who want to redesign their sites, they get very excited about the Standards, and the Federalist platform, because someone has already done the work for them. It makes their job and life a lot easier.
+
+**Standards team:** How did you integrate your work with the Standards?
+
+**Maureen Earley:** We worked with one of 18F’s developers who was doing the various coding elements for us. She redid the back-end of our site to be compliant with the Standards.
+
+**Standards team:** What were the benefits you gained by using the Standards?
+
+**Maureen Earley:** There were several benefits: a) it’s good design, and b) it’s great that we’re contributing to a cohesive branding approach across all of government. Ideally, we will get to a reality that, when you visit a government website, you know you’re on a government site due to the design consistency.
```

Comment:
Also, I asked on `#g-content` what the preferred formatting for this sort of pseudo-list is like, and James Hupp replied with:

> Hmm, we don’t really have anything documented about in-line lists, and I can’t find anything from AP quickly. My own preference is to use `(a)` over `a.` or `a)`. This helps set off the letter from both what comes before and what comes after. If the spacing of the sentence is weird in any way, the version without a leading paren can look a little lost or confusing to me.

I generally agree with this--I prefer `(a)` over `a)`, but it's up to you if you want to change it.